docs/html/reference/requirements-file-format.md

```diff
@@ -74,13 +99,21 @@ and two {ref}`--find-links <install_--find-links>` locations:
 ```
 ````
 
+(per-requirement-options)=
+
 ### Per-requirement options
 
+```{versionadded} 7.0
+
+```
```

Comment:
Eh?

Reply:
What are you confused by?

The version comes directly from the existing text: https://pip.pypa.io/en/stable/cli/pip_install/#per-requirement-overrides

And the empty line is... well... thanks to prettier. 🤷🏽

Reply:
> And the empty line is... well... thanks to prettier. 🤷🏽

🤦‍♂️